repos/fdbt-reference-data-service/src/retrievers/serverless.yml

```diff
@@ -1,199 +1,200 @@
-            BUCKET_NAME: fdbt-csv-ref-data-${self:provider.stage}
-            CONTENT_TYPE: text/csv
-        events:
-            - schedule:
-                  rate: cron(0 2 * * ? *)
-                  enabled: ${self:custom.enableSchedule.${self:provider.stage}, false}
-        alarms:
-            - name: functionDuration
-              threshold: 15000
-              treatMissingData: notBreaching
-            - name: functionThrottles
-              treatMissingData: notBreaching
-            - name: functionErrors
-              treatMissingData: notBreaching
-            - name: functionInvocations
-              treatMissingData: notBreaching
+  NocRetriever:
+    handler: csv_retriever/main.lambda_handler
+    timeout: 60
+    memorySize: 256
+    environment:
+      DATA_URL: https://www.travelinedata.org.uk/wp-content/themes/desktop/nocadvanced_download.php?reportFormat=csvFlatFile&allTable%5B%5D=table_noclines&allTable%5B%5D=table_noc_table&allTable%5B%5D=table_public_name&submit=Submit
+      BUCKET_NAME: fdbt-csv-ref-data-${self:provider.stage}
+      CONTENT_TYPE: text/csv
+    events:
+      - schedule:
+          rate: cron(0 2 * * ? *)
+          enabled: ${self:custom.enableSchedule.${self:provider.stage}, false}
+    alarms:
+      - name: functionDuration
+        threshold: 15000
+        treatMissingData: notBreaching
+      - name: functionThrottles
+        treatMissingData: notBreaching
+      - name: functionErrors
+        treatMissingData: notBreaching
+      - name: functionInvocations
+        treatMissingData: notBreaching
 
-    NaptanRetriever:
-        handler: csv_retriever/main.lambda_handler
-        timeout: 60
-        memorySize: 512
-        environment:
-            DATA_URL: http://naptan.app.dft.gov.uk/DataRequest/Naptan.ashx?format=csv
-            BUCKET_NAME: fdbt-csv-ref-data-${self:provider.stage}
-            CONTENT_TYPE: text/csv
-            TARGET_FILE: Stops.csv
-        events:
-            - schedule:
-                  rate: cron(0 2 * * ? *)
-                  enabled: ${self:custom.enableSchedule.${self:provider.stage}, false}
-        alarms:
-            - name: functionDuration
-              threshold: 25000
-              treatMissingData: notBreaching
-            - name: functionThrottles
-              treatMissingData: notBreaching
-            - name: functionErrors
-              treatMissingData: notBreaching
-            - name: functionInvocations
-              treatMissingData: notBreaching
+  NaptanRetriever:
+    handler: csv_retriever/main.lambda_handler
+    timeout: 60
+    memorySize: 512
+    environment:
+      DATA_URL: https://naptan.dft.gov.uk/Download/File/Stops.csv
+      BACKUP_DATA_URL: https://beta-naptan.dft.gov.uk/Download/File/Stops.csv
```

Comment:
These should be the other way around I think, the beta one is the one that currently works

Reply:
The beta one is the one that works but if we swap them around we will use the beta link until it dies, wouldn't it be better to switch to the normal one as soon as it goes live, which is what the current code will do. Does that make sense?